bindings/java/src/org/sleuthkit/datamodel/FileManager.java

```diff
@@ -91,14 +91,17 @@
 			normalizedPath = normalizedPath + "/";
 		}
 		
-		String query = "SELECT tsk_files.* FROM tsk_files JOIN tsk_objects ON tsk_objects.obj_id = tsk_files.obj_id WHERE parent_path = ? AND name = ? AND data_source_obj_id = ?";
+		String ext = SleuthkitCase.extractExtension(name);
```

Comment:
We should probably not include 'ext' in they query if there isn't an extension. And we shoudl add a comment that this is being done as an optimization